docs/Coiote_DM_Device_Onboarding/Device_Provisioning.md

```diff
@@ -53,24 +48,28 @@ Before running the script some configuration should be set. Example configuratio
 
```

Comment:
> Copy your token, it's valid only for a short period of time. 

I can't comment this part but if it's important maybe we should show this as important part (another line or sth)?

Reply:
done